packages/account/src/account.ts

```diff
@@ -244,21 +244,21 @@ export class Account extends AbstractAccount {
   }
 
   /**
-   * Adds resources to the transaction enough to fund it.
+   * Funds a transaction request by adding the necessary resources.
    *
-   * @param request - The transaction request.
-   * @param requiredQuantities - The coin quantities required to execute the transaction.
-   * @param fee - The estimated transaction fee.
-   * @returns A promise that resolves when the resources are added to the transaction.
+   * @template T - The type of the TransactionRequest.
+   * @param T - request - The transaction request to fund.
+   * @param EstimatedTxParams - The estimated transaction parameters.
```

Comment:
Shouldn't it be like this? 🤔 

```suggestion
   * @param params - The estimated transaction parameters.
```

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2262/commits/76a9375a3849e8adb438f0274c036fe941f6cd50